src/Components/BehaviorsModal/Internal/Widgets/DataHistoryWidget/DataHistoryWidget.tsx

```diff
@@ -1,47 +1,178 @@
-import React, { useContext } from 'react';
```

Comment:
I added mock data history widget data for the story that is used to display widgets in behavior modal: https://explorer.digitaltwins.azure.net/?path=/story/components-behaviors-modal--preview-mode. For individual component we can add stories for all the widget types as a separate PR in the future.

Reply:
Unfortunately that test does not show the chart since it gets cut off. It's good to have for manual validation but can we also pull it out to it's own test so that we get snapshot coverage of it when chromatic turns back on?
![image](https://user-images.githubusercontent.com/57726991/196562213-b26a25ef-35e8-45ef-a2d2-0e70e6512352.png)